site/layouts/partials/bugify.html

```diff
@@ -1,12 +1,8 @@
-{{- $words := (split . " ") -}}
+{{- $words := split . " " -}}
 
 {{- range $word := $words -}}
-  {{- if (strings.Contains $word "#") -}}
```

Comment:
It seems I have opened an issue upstream for this already https://github.com/gohugoio/hugoDocs/issues/739

Even if I find `strings.Contains` more self-explanatory, if it's undocumented, maybe it beats the point. I'm +0 on this change, let me know your thoughts.

Reply:
Doesn't bother me, getting used to Hugo/Go syntax it's still pretty straightforward.